features/conic-gradients.yml.dist

```diff
@@ -13,3 +13,24 @@ status:
     firefox_android: "83"
     safari: "12.1"
     safari_ios: "12.2"
+compat_features:
+  # baseline: false
+  # support:
+  #   chrome: "69"
+  #   chrome_android: "69"
+  #   firefox: "83"
+  #   firefox_android: "83"
+  #   safari: "12.1"
+  #   safari_ios: "12.2"
+  - css.types.image.gradient.conic-gradient
+  - css.types.image.gradient.repeating-conic-gradient
+
+  # baseline: false
+  # support:
+  #   chrome: "72"
+  #   chrome_android: "72"
+  #   firefox: "83"
+  #   firefox_android: "83"
+  #   safari: "12.1"
+  #   safari_ios: "12.2"
+  - css.types.image.gradient.conic-gradient.doubleposition
```

Comment:
Something weird has happened here. How did we get to `baseline: high` without Edge?

Reply:
Filed https://github.com/mdn/browser-compat-data/pull/23603

Reply:
I've sent https://github.com/web-platform-dx/web-features/pull/1294, we shouldn't use both `checkAncestors: true` and `checkAncestors: false` in the dist script.